api/rest/property_value.md

```diff
@@ -6,156 +6,492 @@ parent: REST
+        }
+      ]
     }
+  }
 }
 ```
 
-For each node, `in` contains nodes directed towards the node while `out`
-contains nodes/values directed away from the node. When the property value is a
-node, the result contains node information like `dcid`, `name`, `provenanceId`
-and `types`; when the property value is not a node, it has `value` and
-`provenanceId`.
+### Example 2: Retrieve the addresses of Stuyvesant High School in New York and Gunn High School in California.
+
+#### GET Request
+
+```curl
+curl --request GET \
+  --url 'https://api.datacommons.org/node/property-values?dcids=nces%2F360007702877&dcids=nces%2F062961004587&property=address'
+```
 
-`provenanceId` is the DCID of the provenance for the corresponding value.
+#### POST Request
+
+```curl
+curl --request POST \
+  --url https://api.datacommons.org/node/property-values \
+  --header 'content-type: application/json' \
+  --data '{
+	"dcids": [
+		"nces/360007702877",
+		"nces/062961004587"
+	],
+	"property": "address"
+}'
+```
 
-<!--- TODO: add link to the data model --->
+#### Response
 
-**NOTE:** Please run `JSON.parse()` on the `payload` field to retrieve the data.
-For example, in JavaScript: `var data = JSON.parse(response['payload'])`.
+```json
+{
+  "payload": {
+    "nces/062961004587": {
+      "out": [
+        {
+          "provenanceId": "dc/mzy8we",
+          "value": "780 Arastradero Rd., Palo Alto, California"
+        }
+      ]
+    },
+    "nces/360007702877": {
+      "out": [
+        {
+          "provenanceId": "dc/mzy8we",
+          "value": "345 Chambers St, New York, New York"
+        }
+      ]
+    }
+  }
+}
+```
 
-## Error Response
+### Example 3: Retrieve a list of earthquakes in Madagascar.
 
-### **Code**: `500 Internal Server Error`
+#### GET Request
 
-**Request example:** (property not specified)
+```curl
+curl --request GET \
+  --url 'https://api.datacommons.org/node/property-values?dcids=country%2FMDG&property=affectedPlace&valueType=EarthquakeEvent'
+```
 
-```bash
-curl -X POST 'https://api.datacommons.org/node/property-values' -d '{"dcids": ["geoId/06"]}'
+#### POST Request
+
+```curl
+curl --request POST \
+  --url https://api.datacommons.org/node/property-values \
+  --header 'content-type: application/json' \
+  --data '{
+	"dcids": [
+		"country/MDG"
+	],
+	"property": "affectedPlace",
+	"valueType": "EarthquakeEvent"
+}'
 ```
 
-**Response content example**
+#### Response
 
 ```json
 {
-  "code": 2,
-  "message": "missing required arguments"
+  "payload": {
+    "country/MDG": {
+      "in": [
+        {
+          "dcid": "earthquake/usp000jgbb",
+          "name": "Madagascar",
+          "provenanceId": "dc/xz8ndk3",
+          "types": [
+            "EarthquakeEvent"
+          ]
+        },
+        {
+          "dcid": "earthquake/usp000h6zw",
+          "name": "Madagascar",
+          "provenanceId": "dc/xz8ndk3",
+          "types": [
+            "EarthquakeEvent"
+          ]
+        },
+        {
```

Comment:
I'm not sure if it's illustrative to include the entire example here. Could we use ellipses like you did in the blocks above?